pkg/azure/sdk_handler.go

```diff
@@ -548,6 +551,7 @@ func (h *AzureSDKHandler) CreateParagliderVirtualNetwork(ctx context.Context, lo
 			},
 		},
 	}
+	h.createParagliderNamespaceTag(&parameters.Tags)
```

Comment:
I am not opposed to this tagging way of doing things, but before our standard has been to determine the namespace of a resource based on what vnet it is in / a naming convention. Is there a reason that doesn't work here?

Reply:
It just seemed like more than it's worth to find all resources created for a test. It also wouldn't cover resources like the vnet that the VPN gateway resides in (at the very least there would be a lot of pointer chasing). I felt this was the most straightforward way to do this in the fastest time.